homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -0,0 +1,72 @@
+"""Platform for Husqvarna Automower basic entity."""
+
+import datetime
+import logging
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import dt as dt_util
+
+from . import AutomowerDataUpdateCoordinator
+from .const import DOMAIN, HUSQVARNA_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AutomowerEntity(CoordinatorEntity[AutomowerDataUpdateCoordinator]):
+    """Defining the Automower Basic Entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, coordinator, idx) -> None:
```

Comment:
```suggestion
    def __init__(self, coordinator: AutomowerDataUpdateCoordinator, idx: int) -> None:
```
We love a well typed integration <3